networking/network_policy/multitenant-network-policy.adoc

```diff
@@ -1,7 +1,12 @@
 :_content-type: ASSEMBLY
 [id="multitenant-network-policy"]
 = Configuring multitenant isolation with network policy
+ifdef::openshift-enterprise[]
```

Comment:
The same comments apply here regarding the attribute files. Please see my preceding comment for details.